### ISSUE TITLE
Discuss with add-ons developer for strings to be included by default

### ISSUE THREAD

**Issue by AntoineTurmel** [nightingale-media-player/nightingale-l10n#9]
Some add-ons could benefit having strings directly in Nightingale l10n like Playlists folder.
We should discuss that, and maybe checkout most used add-ons and review which strings are most used.

**Comment by rsjtdrjgfuzkfg** [comment]
For me: Playlist folders (as it is core), Service Pane headlines (Radio, Services, others to come?) and maybe a add-on section in the help menu as MLyrics uses it ("Add-on Help"). Not quite sure about what else is a cross-add-on requirement...

**Comment by AntoineTurmel** [comment]
@rsjtdrjgfuzkfg maybe I could request some statistics from Babelzilla to check what are the most included strings for Nightingale/Songbird add-ons.

**Comment by rsjtdrjgfuzkfg** [comment]
@GeekShadow nice idea :)

**Comment by AntoineTurmel** [comment]
Ok, Fenian from Babelzilla was able to give us a complete output text file with all strings from Songbird/Nightingale add-ons :
http://www.babelzilla.org/forum/index.php?showtopic=7242&view=findpost&p=63087

We should analyze what is the top strings used and if it's worth to include some.

**Comment by AntoineTurmel** [comment]
@rsjtdrjgfuzkfg so can I integrate those two strings ?

https://github.com/nightingale-media-player/nightingale-hacking/blob/sb-trunk-oldxul/extensions/playlistfolders/chrome/locale/en-US/playlistfolders.properties

**Comment by rsjtdrjgfuzkfg** [comment]
@GeekShadow: yes, would be great to have them in core. If threads are ported to the recent xulrunner, we could think of including FolderSync strings as well.
